css/style.scss

```diff
@@ -605,13 +605,10 @@
 
       img {
         display: block;
-        max-height: 99%;
+        max-height: 100%;
         position: absolute;
         top: 0; bottom:0; left: 0; right:0;
         margin: auto;
-        //min-height: 100%;
-        //min-width: 100%;
-        //position: absolute;
```

Comment:
Nice! Thanks for fixing this. I was just trusting you knew what was going on, because I didn't :laughing: